lib/pbench/test/unit/server/test_datasets_detail.py

```diff
@@ -2,6 +2,23 @@
 import re
 import requests
 from http import HTTPStatus
+from requests.exceptions import HTTPError
+
+
+def make_http_exception(status: int) -> HTTPError:
```

Comment:
Seems like this should land in a common module for the tests.

Reply:
I'd like to refactor all the `query_api` tests into a class hierarchy like I did on the implementation side; I just keep putting that off for a separate PR I haven't yet gotten to. Aside from that, I'm not entirely sure where to put it. It can't really be a fixture, so it doesn't really belong in `conftest.py`, but it doesn't quite belong anywhere else, either.